spec/controllers/blob_controller_spec.rb

```diff
@@ -45,7 +45,10 @@
 
     context 'redirect to tree' do
       let(:id) { 'markdown/doc' }
-      it { should redirect_to("/#{project.path_with_namespace}/tree/markdown/doc") }
+      it "redirects" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.